environment/src/main/java/com/sequenceiq/environment/environment/v1/converter/EnvironmentApiConverter.java

```diff
@@ -395,6 +397,13 @@ public UpdateAzureResourceEncryptionDto convertUpdateAzureResourceEncryptionDto(
                 .build();
     }
 
+    public UpdateAwsDiskEncryptionParametersDto convertUpdateAwsDiskEncryptionParametersDto(UpdateAwsDiskEncryptionParametersRequest request) {
+        return UpdateAwsDiskEncryptionParametersDto.builder()
+                .withAwsDiskEncryptionParametersDto(
+                        awsDiskEncryptionParametersToAwsDiskEncryptionParametersDto(request.getAwsDiskEncryptionParameters()))
```

Comment:
I think `request.getAwsDiskEncryptionParameters() == null` is possible, in which case `awsDiskEncryptionParametersToAwsDiskEncryptionParametersDto()` will throw an NPE. The logic there should be fixed to tolerate this by returning a DTO instance that has a `null` `encryptionKeyArn`.

Reply:
Made the changes

Reply:
Addressed.